src/components/reports/formSteps/Variant.tsx

```diff
@@ -0,0 +1,80 @@
+import { FC } from "react";
+
+import FieldSet from "../FieldSet";
+import FormStepBtn from "../../UI/FormStepBtn";
+import classes from "./Variant.module.css";
+
+interface Props {
+  nextStep: () => void;
+  prevStep: () => void;
+  variantExists: boolean;
+  setVariantExists: (bool: boolean) => void;
+  setFieldValue: (field: string, value: string | string[] | boolean) => void;
+}
+
+const Variant: FC<Props> = (props) => {
```

Comment:
we should also allow adding of multiple variants, will create an issue for this!